test.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require '../functions.php';
```

Comment:
incorrect file path - also test needs to be in a folder

Reply:
agreed, will put in a 'test' folder in 'src'?

Reply:
perfect!